specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/examples/CreateAndUpdateTestRun.json

```diff
@@ -0,0 +1,171 @@
+{
+  "parameters": {
+    "testRunId": "12316678-1234-1234-1234-122451189012",
+    "oldTestRunId": "12345678-1234-1234-1234-123456789012",
+    "api-version": "2022-06-01-preview",
+    "body": {
+      "displayName": "Performance_LoadTest_Run1",
+      "testId": "12345678-1234-1234-1234-123456789012",
+      "description": "sample description",
+      "loadTestConfig": {
+        "engineInstances": 6,
+        "splitAllCSVs": true
+      },
+      "passFailCriteria": {
+        "passFailMetrics": {
+          "fefd759d-7fe8-4f83-8b6d-aeebe0f491fe": {
+            "clientmetric": "response_time_ms",
+            "aggregate": "percentage",
+            "condition": ">",
+            "value": 20,
+            "action": "continue"
+          }
+        }
+      },
+      "testArtifacts": {
+        "outputArtifacts": {
+          "resultUrl": {
+            "url": "https://dummyurl.com/dummyresourceresult",
+            "fileId": "92345678-1234-1111-1234-123456789012",
+            "filename": "sample.jmx",
+            "fileType": 2,
```

Comment:
we can change it to 0

Reply:
1 is for user properties and 2 for any other file, right?

Reply:
0 is for the JMX script recently

Reply:
Has it been deployed?
I still see 2 in the portal.

Reply:
not yet but we can change it

Reply:
Ok i will update it